docs/pages/admin-guides/access-controls/sso/sso.mdx

```diff
@@ -411,6 +411,103 @@ values to match your identity provider:
 
 At this time, the `spec.provider` field should not be set for any other identity providers.
 
+## Configuring SSO for MFA checks
+
+Teleport administrators can configure Teleport to delegate MFA checks to an 
+SSO provider as an alternative to registering MFA devices directly with the Teleport cluster.
+This allows Teleport users to use MFA devices and custom flows configured in the SSO provider
+to carry out privileged actions in Teleport, such as:
+
+- MFA device management (e.g. register or delete a webauthn device in Teleport)
+- [Per-session MFA](../guides/per-session-mfa.mdx)
+- [Moderated sessions](../guides/moderated-sessions.mdx)
+- [Admin actions](../guides/mfa-for-admin-actions.mdx)
+
+Administrators may want to consider enabling this feature for the following benefits:
+
+- All authentication (login and MFA) goes through the IDP, reducing administrative overhead
+- Make custom MFA flows, such as prompting for 2 distinct devices for a single MFA check
+- Integrate with non-webauthn devices supported directly by your IDP
+
+<Admonition type="warning">
+  SSO MFA is an enterprise feature. Only OIDC and SAML auth connectors are supported.
+</Admonition>
+
+### Configure the IDP App / Client
+
+There is no standardized MFA flow unlike there is with SAML/OIDC login, so
+each IDP may offer zero, one, or more ways to offer MFA checks.
+
+Teleport does not make any assumptions as to how the MFA app is configured.
+If desired, you could even use your basic login flow with username, password,
+and MFA device.
```

Comment:
the example yaml's seem to suggest the user should create a separate app in the IDP for doing MFA, should we be more explicit about suggesting that in this section? It feels like we're not really telling people what they need to do here

Reply:
This is intentionally left vague and meant as an exercise for the reader to set up their own specific solution. You could set up a separate app or use the same app depending on the IDP offerings.

You're right it is too vague though, I will leave a couple examples and more guidance. My only concern is that I haven't actually been able to test these approaches with MFA enabled, since we don't have access to an enterprise Okta or Auth0 account for testing.